doc/read-the-docs-site/tutorials/AuctionValidator.hs

```diff
@@ -193,5 +193,5 @@ auctionValidatorScript ::
   CompiledCode (BuiltinData -> BuiltinData -> BuiltinData -> ())
 auctionValidatorScript params =
   $$(PlutusTx.compile [||auctionUntypedValidator||])
-    `PlutusTx.unsafeApplyCode` PlutusTx.liftCode params
+    `PlutusTx.unsafeApplyCode` PlutusTx.liftCodeDef params
```

Comment:
Should we do this in the tutorial? This won't work if someone actually tries to run this on testnet, does it? Because plutus core 1.1.0 hasn't been allowed yet.

Reply:
Well, they won't be able to use these versions of the libraries either! But yes, I guess we can include an explicit version here.